tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature

```diff
@@ -1922,3 +1918,633 @@ Feature: listing sharedWithMe when auto-sync is disabled
+    Given using spaces DAV path
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "to share" to "textfile.txt"
+    And user "Alice" has sent the following share invitation:
+      | resource        | textfile.txt |
+      | space           | projectSpace |
+      | sharee          | grp1         |
+      | shareType       | group        |
+      | permissionsRole | File Editor  |
+    And user "Alice" has sent the following share invitation:
+      | resource        | textfile.txt |
+      | space           | projectSpace |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+        {
+          "type": "object",
+          "required": ["value"],
+          "properties": {
+            "value": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "required": [
+                  "@UI.Hidden",
+                  "@client.synchronize",
+                  "eTag",
+                  "file",
+                  "id",
+                  "lastModifiedDateTime",
+                  "name",
+                  "parentReference",
+                  "remoteItem",
+                  "size"
+                ],
+                "properties": {
+                  "@UI.Hidden": {
+                    "type": "boolean",
+                    "enum": [false]
+                  },
+                  "@client.synchronize": {
+                    "type": "boolean",
+                    "enum": [false]
+                  },
+                  "eTag": {
+                    "type": "string",
+                    "pattern": "%etag_pattern%"
+                  },
+                  "file": {
+                    "type": "object",
+                    "required": [
+                      "mimeType"
+                    ],
+                    "properties": {
+                      "mimeType": {
+                        "type": "string",
+                        "enum": ["text/plain"]
+                      }
+                    }
+                  },
+                  "id": {
+                    "type": "string",
+                    "pattern": "^%share_id_pattern%$"
+                  },
+                  "name": {
+                    "type": "string",
```

Comment:
Try to minimize the line numbers of a schema by removing unnecessary checks. (The required key is sufficient for some checks) Refer to other added scenarios.

Reply:
I have try to minimized can you check if that is ok or not  @grgprarup